src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+const { REACT_APP_API_URL } = process.env
+
+export function getProtocolsList() {
+  return axios
+    .get(`${REACT_APP_API_URL}/clinic/protocols`)
+    .then((res) => res.data.protocols)
+    .catch((err) => console.error(err))

```

Comment:
don't catch the error from here, catch it from the component so you can show something to the user in case something happens